src/widgets/helper/QColorPicker.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
Please include the original license from the files these were copied from. explain the difference as you explained it below

Reply:
See 0d51d90.